code/src/Components/Drink.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+export const Drink = ({ drink, setDrink }) => {
+  const handleDrinkChange = (event) => {
+    setDrink(event.target.value);
+  }
+
+  return (
+    <form>
+      <div className="drinkComponent">
+        <h2>What drink would you like as a reward for surviving another day of coding?</h2>
+
+        <select
+          onChange={handleDrinkChange}
+          className="selectDropdown"
+          aria-label="Select drink"
+          tabIndex="0"
```

Comment:
Like we talked about on the Monday meeting, I am so far from an expert in the area of accessibility etc. But I don't think that you need to use a tab-index here, since the Select is a native HTML element which automatically is targeted when using tab (just like buttons, links and input elements for example). The only reason to use a tab index here, would be to change the order in which the site is "tab-able". But from what I can see, that should not be necessary 🥳